### PR TITLE
add amip requirements to docs

### LINKED ISSUE #652: add checklist of AMIP requirements to docs
Before our release, we should make sure that our docs contain everything we want them to, including a checklist of AMIP requirements

### PR DESCRIPTION
**Pull request by szy21**
## Purpose 

Closes #652 

## Content

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by szy21** [comment]
@juliasloan25 @LenkaNovak Here is a draft of a brief amip description. I wonder:
1. Would you like it to be more detailed?
2. Would you like to have references?
3. For components with `None`, would you like them to be removed?

File: docs/src/index.md [hunk 1]
```diff
@@ -24,6 +24,51 @@ or coupling components (using MPI) that are running on different processes.
...
+Orographic gravity wave drag: None
+
+Non-orographic gravity wave drag: None
+
+Aerosols and chemistry: None
+
+### Land
+
+Dynamical core:
+- Equation: Following Manabe bucket hydrology scheme
```
Comment (juliasloan25) [inline docs/src/index.md:55]: Could be good to add a citation here!

File: docs/src/index.md [hunk 2]
```diff
@@ -24,6 +24,51 @@ or coupling components (using MPI) that are running on different processes.
...
+- Prognostic variables: Temperature, water content, snow water content
+- Spatial discretization: Finite difference, single column
```
Comment (juliasloan25) [inline docs/src/index.md:57]: is single column correct? I thought for global runs we have many columns over the sphere

**Review by juliasloan25 — COMMENTED** [review]
This looks good to me, Zhaoyi! Now that I see the details, this might fit better in the experiments/AMIP/README.md file since that's where the AMIP driver lives and the coupler supports other setups too.

File: docs/src/index.md (hunk 2 above)
Comment (szy21) [reply docs/src/index.md:57]: Ah, yes, I meant there is no horizontal transport between each column. I'll think about a better description.

**Review by LenkaNovak — COMMENTED** [review]
Looks great, thank you, @szy21 ! 

Shall we add more details on insolation and EDMF? I've been notionally thinking something like [this](https://pcmdi.llnl.gov/mips/amip/home/Documentation/20gfdl.html#RTFToC2), but maybe being somewhat less specific on the details that are changing is fine. :)

File: docs/src/index.md [hunk 3]
```diff
@@ -24,6 +24,51 @@ or coupling components (using MPI) that are running on different processes.
...
+- Time stepping: Explicit additive Runge–Kutta method
+
+Land surface albedo: 
+- Bare ground: Prescribed from files
+- Snow: Constant
```
Comment (LenkaNovak) [inline docs/src/index.md:62]: There is a simple prognostic snow in the bucket model.

**Comment by juliasloan25** [comment]
to answer your questions...

> Would you like it to be more detailed?
Would you like to have references?
For components with None, would you like them to be removed?

1. I think the detail level is good, but I'm less familiar with the physics so will default to you two on that
2. References would be nice if possible, but if we don't have them it's okay too
3. To me it seems like it's good to keep "none" for components we don't have so it's clear what is and isn't included

**Comment by szy21** [comment]
hmm for some reason my branch is updated but this PR is not. I'll try close and reopen.

**Pull request closed without merging**